OpenRA.Mods.Common/Traits/World/EditorActionManager.cs

```diff
@@ -31,6 +31,7 @@ public class EditorActionManager : IWorldLoaded
 		int nextId;
 
 		public bool Modified;
+		public bool SaveFailed;
 
 		public void WorldLoaded(World w, WorldRenderer wr)
```

Comment:
Would it make sense to add `Modified = false;` after `Add(new OpenMapAction());`?

Currently `Modified` is set to true on load, which just seems incorrect

Reply:
Why is Modified set to true on load in the first place?